20176.0.txt

```diff
@@ -0,0 +1,59 @@
+@0
+
+
+@1
+# Put IN.A into memory
+MOV UP, ACC
+SAV
+# Subtract IN.B from IN.A
+SUB RIGHT
+SWP
+# Send IN.A to be subtracted from IN.B
```

Comment:
Worth commenting that the `SAV` and `SWP` ensure that you still have the value of `IN.A` at this point